<launch>
  <!-- Include some relevant files  -->
  <include file="$(find swiftpro_control)/launch/inc/swiftpro_spawner.xml"/>
  <include file="$(find swiftpro_control)/launch/inc/red_box_spawner.xml"/>
  <include file="$(find swiftpro_control)/launch/inc/robot_control.xml"/>

  <!-- Launch Gazebo World -->
  <!-- World File-->
  <arg name="world_file" default="$(find swiftpro_control)/world/empty.world"/>

  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="use_sim_time" value="true"/>
    <arg name="debug" value="false"/>
    <arg name="gui" value="true" />
    <arg name="world_name" value="$(arg world_file)"/>
  </include>
  
   <!-- call fk_swiftpro service-->
  <node name="fk_swiftpro" pkg="swiftpro_control" type="fk_swiftpro"/>

  <!-- call fk_swiftpro service-->
  <node name="ik_swiftpro" pkg="swiftpro_control" type="ik_swiftpro"/>

</launch>
